### ISSUE TITLE
switch tests to pytest

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/vak#87]
+ make setup use and generate test configs
  - i.e. have configs named 
+ add config files for 'audio_wav_annot_koumura'
+ fix config output paths to match this naming convention also, so structure of test_data is like
```
test_data/
  prep/
    train/
      audio_cbin_annot_notmat/
      audio_wav_annot_koumura/
      ...
    predict/
      audio_cbin_annot_notmat/
      audio_wav_annot_koumura/
      ...
  results/
    train/
      audio_cbin_annot_notmat/
      audio_wav_annot_koumura/
      ...
    predict/
      audio_cbin_annot_notmat/
      audio_wav_annot_koumura/
      ...
```
+ naming convention for fixtures should match that used with configs
  - denoting data source (audio / spect) and annotation type

**Comment by NickleDave** [comment]
+ just move everything in `unit_tests` up a level because there's not actually anything in `integration_tests` right now anyway

**Comment by NickleDave** [comment]
fixed by #309
